endpoints/0-persistence-endpoints.md

```diff
@@ -0,0 +1,95 @@
+#### 0.1 - Clean All Endpoint
+**Description**: Cleans all clusters, clouds, and edge devices in the SAL and undeploys them from the ProActive server.
+Note that all asynchronous processes related to cloud node candidate synchronization should be completed; otherwise, clouds will not be removed. To validate, please call the [isAnyAsyncNodeCandidatesProcessesInProgress](https://github.com/ow2-proactive/scheduling-abstraction-layer/blob/master/endpoints/2-cloud-endpoints.md#23--isanyasyncnodecandidatesprocessesinprogress-endpoint) endpoint.
+
+
+**Path:**
+
+```url
+🟡 DELETE {{protocol}}://{{sal_host}}:{{sal_port}}/sal/persistence/cleanall
```

Comment:
I think it would be better to have endpoint defined as:

`{{protocol}}://{{sal_host}}:{{sal_port}}/sal/persistence/clean/all (or just /clean)  
`{{protocol}}://{{sal_host}}:{{sal_port}}/sal/persistence/clean/clouds`
`{{protocol}}://{{sal_host}}:{{sal_port}}/sal/persistence/clean/clusters`
`{{protocol}}://{{sal_host}}:{{sal_port}}/sal/persistence/clean/edges`
`{{protocol}}://{{sal_host}}:{{sal_port}}/sal/persistence/clean/SALdatabase`

Reply:
yes you are right, updating